.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -1,29 +1,59 @@
 parameters:
+  aroVersionStorageAccount: ''
+  azureDevOpsE2EJSONSPN: ''
   location: ''
+  rpMode: ''
   subscription: ''
-  azureDevOpsE2EJSONSPN: ''
 
 steps:
-# TODO(mj) we need to checkout code to run tooling like kubeconfig. This should go away once we refactor everything into golang.
-- template: ./template-checkout.yml
-- template: ./template-az-cli-login.yml
-  parameters:
-    azureDevOpsJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
 - script: |
     set -e
     export LOCATION=${{ parameters.location }}
     export AZURE_SUBSCRIPTION_ID=${{ parameters.subscription }}
 
-    trap 'rm -f devops-spn.json' EXIT
+    trap 'rm -f devops-spn.json ; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af; rm -rf ~/.docker/config.json' EXIT
     base64 -d >devops-spn.json <<<${{ parameters.azureDevOpsE2EJSONSPN }}
     export AZURE_CLIENT_ID=$(jq -r .clientId <devops-spn.json)
     export AZURE_CLIENT_SECRET=$(jq -r .clientSecret <devops-spn.json)
     export AZURE_TENANT_ID=$(jq -r .tenantId <devops-spn.json)
 
+
+    # TODO: need to fetch $IMAGE and $ARO_VERSION_SA, ideally from the RP
+    # config.  Would be good to have the original git commit of the RP config
+    # and use it.
+    # TODO: We are testing using INT registry as production would require to grant access
+    # to different tenant.
+    export IMAGE=arointsvc.azurecr.io/aro
+    docker login $IMAGE -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET
+
     set -x
-    . ./hack/e2e/run-rp-and-e2e.sh
 
-    export CI=true
-    run_e2e
+    VERSION=$(curl -sf "https://${{ parameters.aroVersionStorageAccount }}.blob.core.windows.net/rpversion/$LOCATION")
+
+    # TODO: consider removing E2E_CREATE_CLUSTER, E2E_DELETE_CLUSTER below after next RP
+    # deployment.
+
+    # TODO: Remove AZURE_FP_CLIENT_ID once RP is deployed in INT
+    export AZURE_FP_CLIENT_ID="8aa2578d-0948-477f-b157-02d3fce19fa0"
```

Comment:
check vs 71cfb175-ea3a-444e-8c03-b119b2752ce4 ?

Reply:
What is the true source of data here? I think 8aa is right one... `aro-v4-e2e-fp-shared`

Reply:
ok, we use one with wrong name :/